sdl2promela/stop_condition/translator.py

```diff
@@ -490,6 +490,61 @@ def _generate_exist(context: GenerateContext, expr: model.CallExpression):
     )
 
 
+def _get_queue_name(context: GenerateContext, expr: model.CallExpression):
```

Comment:
As it is now, it is hard to tell what is the purpose of this function. Could you please rename it and provide some comments?

Reply:
Renamed